tensornetwork/backends/jax/jax_backend.py

```diff
@@ -402,3 +402,12 @@ def expm(self, matrix: Tensor):
 
   def jit(self, fun: Callable, *args: List, **kwargs: dict) -> Callable:
     return libjax.jit(fun, *args, **kwargs)
+
+  def sum(self, tensor: Tensor, axis: Optional[Sequence[int]] = None) -> Tensor:
```

Comment:
Do these methods exist in the BaseBackend?

Reply:
give me one sec, this was accidentially added, fixing it right now

Reply:
OK, should be good to go!